LayoutTests/accessibility/display-contents/descendant-menu-item-expected.txt

```diff
@@ -1,10 +1,124 @@
-This test ensures that the aria-activedescendant of a display:contents menu is considered selected.
-
-PASS: accessibilityController.accessibleElementById('item1').isSelected === true
-PASS: accessibilityController.accessibleElementById('item2').isSelected === false
-
-PASS successfullyParsed is true
-
-TEST COMPLETE
-Menu item 1
-Menu item 2
+CONSOLE MESSAGE: ReferenceError: Can't find variable: platformValueForW3CName
```

Comment:
This output indicates the test is probably broken

Reply:
Ah, good catch. I had the script look for the word FAIL in the updated expectations. I modified it to search for CONSOLE also and exclude those from the upload.

The good news is that 80% of the tests it tried to modify did seem to pass still